.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -21,8 +21,8 @@ Fixes # [ISSUE]
 
 
 ### How Has This Been Tested?
-Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.
 
+Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test. Also, add the build link to the Github Actions Workflow (e.g. https://github.com/anitab-org/stem-diverse-tv/actions/runs/747608423)
```

Comment:
I don't think we need this change. Whether CI builds are successful or not can be seen easily in pull request